src/components/DiagnosticsBody.vue

```diff
@@ -102,6 +107,24 @@ export default Vue.extend({
     toggleMute() {
       this.hideMuted = !this.hideMuted;
     },
+    updateExpandVisibility(group: DiagnosticsCategory) {
+      const expanded = this.expanded[group];
+      const groupRow = this.$refs[`group-${ group }`];
+
+      if (groupRow instanceof HTMLElement) {
+        groupRow.setAttribute('aria-expaneded', expanded ? 'true' : 'false');
```

Comment:
Looks like a typo: 'aira-expaneded' -- should be `'aria-expaneded'` ?

Reply:
Fixing this didn't fix the toggle problem I wrote up (but I didn't do a full restart).